src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_credential_free.py

```diff
@@ -821,7 +821,7 @@ def get_connection_string(self):
             'az account get-access-token --resource-type oss-rdbms').get('accessToken')
 
         # extension functions require the extension to be available, which is the case for postgres (default) database.
-        conn_string = "host={} user={} dbname=postgres password={} sslmode=require".format(
+        conn_string = "host={} user='{}' dbname=postgres password={} sslmode=require".format(
```

Comment:
Cloud the db password also contain blank space since it's raw password not hashed? rare case tho, user set so.

Reply:
Solved. The password is accessKey not user set, has no blankspace.